docs/data-sources/lbs.md

```diff
@@ -0,0 +1,55 @@
+---
+page_title: "Scaleway: scaleway_lbs"
+description: |-
+Gets information about multiple Load Balancers.
+---
+
+# scaleway_lbs
+
+Gets information about multiple Load Balancers.
+
+## Example Usage
+
+```hcl
+# Find LBs by name
+data "scaleway_lbs" "my_key" {
+  name = "foobar"
```

Comment:
Please add support for tags, it will help to select several load balancers in one go.

Reply:
Tags support will be added afterward when its featured on api side